[2022-06-02 12:36:48,416] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: python_dag_with_jinja.print_current_date_with_jinja scheduled__2022-06-01T00:30:00+00:00 [queued]>
[2022-06-02 12:36:48,424] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: python_dag_with_jinja.print_current_date_with_jinja scheduled__2022-06-01T00:30:00+00:00 [queued]>
[2022-06-02 12:36:48,425] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2022-06-02 12:36:48,425] {taskinstance.py:1242} INFO - Starting attempt 1 of 2
[2022-06-02 12:36:48,425] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-06-02 12:36:48,435] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): print_current_date_with_jinja> on 2022-06-01 00:30:00+00:00
[2022-06-02 12:36:48,438] {standard_task_runner.py:52} INFO - Started process 71039 to run task
[2022-06-02 12:36:48,443] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'python_dag_with_jinja', 'print_current_date_with_jinja', 'scheduled__2022-06-01T00:30:00+00:00', '--job-id', '33', '--raw', '--subdir', '/Users/philhoonoh/Desktop/product_serving/Boostcamp-AI-Tech-Product-Serving/test_airflow/dags/04-python-operator-with-jinja.py', '--cfg-path', '/var/folders/46/94mtqk_n4x74p14rc4c63cnr0000gn/T/tmppg48cde1', '--error-file', '/var/folders/46/94mtqk_n4x74p14rc4c63cnr0000gn/T/tmp830kxq7c']
[2022-06-02 12:36:48,445] {standard_task_runner.py:77} INFO - Job 33: Subtask print_current_date_with_jinja
[2022-06-02 12:36:53,476] {logging_mixin.py:109} INFO - Running <TaskInstance: python_dag_with_jinja.print_current_date_with_jinja scheduled__2022-06-01T00:30:00+00:00 [running]> on host PHILui-MacBook-Pro.local
[2022-06-02 12:36:58,504] {logging_mixin.py:109} WARNING - /Users/philhoonoh/Desktop/product_serving/Boostcamp-AI-Tech-Product-Serving/test_airflow/.venv/lib/python3.9/site-packages/airflow/models/xcom.py:322 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-06-02 12:36:58,513] {logging_mixin.py:109} WARNING - /Users/philhoonoh/Desktop/product_serving/Boostcamp-AI-Tech-Product-Serving/test_airflow/.venv/lib/python3.9/site-packages/airflow/models/renderedtifields.py:126 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-06-02 12:36:58,521] {logging_mixin.py:109} WARNING - /Users/philhoonoh/Desktop/product_serving/Boostcamp-AI-Tech-Product-Serving/test_airflow/.venv/lib/python3.9/site-packages/airflow/models/renderedtifields.py:162 SAWarning: Coercing Subquery object into a select() for use in IN(); please pass a select() construct explicitly
[2022-06-02 12:36:58,525] {logging_mixin.py:109} WARNING - /Users/philhoonoh/Desktop/product_serving/Boostcamp-AI-Tech-Product-Serving/test_airflow/.venv/lib/python3.9/site-packages/airflow/models/renderedtifields.py:159 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-06-02 12:36:58,531] {taskinstance.py:1412} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=philhoon
AIRFLOW_CTX_DAG_ID=python_dag_with_jinja
AIRFLOW_CTX_TASK_ID=print_current_date_with_jinja
AIRFLOW_CTX_EXECUTION_DATE=2022-06-01T00:30:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-01T00:30:00+00:00
[2022-06-02 12:36:58,540] {logging_mixin.py:109} INFO - 2022-06-01는 수요일입니다
[2022-06-02 12:36:58,541] {python.py:152} INFO - Done. Returned value was: None
[2022-06-02 12:36:58,550] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=python_dag_with_jinja, task_id=print_current_date_with_jinja, execution_date=20220601T003000, start_date=20220602T033648, end_date=20220602T033658
[2022-06-02 12:36:58,578] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-06-02 12:36:58,599] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-02 12:36:58,602] {logging_mixin.py:109} WARNING - /Users/philhoonoh/Desktop/product_serving/Boostcamp-AI-Tech-Product-Serving/test_airflow/.venv/lib/python3.9/site-packages/airflow/models/dag.py:1069 SADeprecationWarning: Query.value() is deprecated and will be removed in a future release.  Please use Query.with_entities() in combination with Query.scalar() (deprecated since: 1.4)
[2022-06-02 12:36:58,603] {logging_mixin.py:109} WARNING - /Users/philhoonoh/Desktop/product_serving/Boostcamp-AI-Tech-Product-Serving/test_airflow/.venv/lib/python3.9/site-packages/airflow/models/dag.py:1069 SADeprecationWarning: Query.values() is deprecated and will be removed in a future release.  Please use Query.with_entities() (deprecated since: 1.4)
